### PR TITLE
RPRBLND-2257: Math node provides error

### PR DESCRIPTION
**Pull request by VascoPi**
### PURPOSE
Math node provides an error.

### EFFECT OF CHANGE
Fixed error 0.0 cannot be raised to a negative power.
Added support for the math operations: Truncate, Square Root, Inverse Square Root, Sign, Ping Pong.

### TECHNICAL STEPS
Sanitize data for POWER operation.
Added support for operation TRUNC, SQRT, INVERSE_SQRT, SING, and PINGPONG.

### CODE REVIEW

File: src/rprblender/nodes/blender_nodes.py [hunk 1]
```diff
@@ -1504,7 +1518,7 @@ def export(self):
         if self.node.use_clamp:
             res = res.clamp()
 
-        return res
+        return res.to_bw()
```
[the remark this answers is not in the document]
Comment (VascoPi) [reply src/rprblender/nodes/blender_nodes.py]: Improved https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderBlenderAddon/pull/595/commits/d6ba66dfb5e1d03698080175f8ad2e7e6e114a12

File: src/rprblender/nodes/blender_nodes.py [hunk 2]
```diff
@@ -1428,7 +1428,8 @@ class ShaderNodeMath(NodeParser):
 
     def export(self):
         op = self.node.operation
-        in1 = self.get_input_value(0)
+        # we use a value data from the input
+        in1 = self.get_input_value(0).rgb_to_hsv().get_channel(2)
```
Comment (bnagirniak) [inline src/rprblender/nodes/blender_nodes.py:1432]: Compare this with cycles. This has to be checked.

Comment (VascoPi) [reply src/rprblender/nodes/blender_nodes.py]: reverted https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderBlenderAddon/pull/595/commits/552598c82734de2878afcb3acf6b3d8d0cc5efb0

**Review by DagerD — APPROVED** [review]

**Review by bnagirniak — APPROVED** [review]

**Review by bnagirniak — APPROVED** [review]

**Review by ARozghon — APPROVED** [review]
Approved by QA

**Pull request merged**